source/releasenotes/2024-03-18-PHP-7-1-EOS.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "PHP versions below 7.2 to reach end-of-sale"
+published_date: "2024-03-18"
+categories: [infrastructure, security, action-required]
+---
+As part of our continued effort to provide the latest and best in secure software, PHP versions 7.1 and below will reach end-of-sale (EoS) on May 15, 2024. This means that sites created after May 15 will not be able to change their PHP version to PHP 7.1, 7.0, or any version of PHP 5. Sites created with custom upstreams using EoS PHP will also fail to deploy during site creation.
+
+Sites already running PHP 5, PHP 7.0, or PHP 7.1 will continue to run even after May 15.
+
+PHP 7.1 was declared end-of-life (EoL) by [the PHP organization](https://www.php.net/supported-versions.php) on November 30, 2020, more than three years ago. PHP 5.6 reached EoL on December 31, 2018, more than five years ago. EoL software does not receive security or feature updates, and could expose sites to attack if any vulnerabilities or exploits are discovered.
+
+**Action required**
+
+Customers using custom upstreams with a PHP version less than 7.2 should update their custom upstreams by May 15 to avoid disruption. The current supported versions of PHP are 8.1, 8.2, and 8.3. Pantheon [currently recommends](https://docs.pantheon.io/guides/php#supported-php-versions) at least PHP 8.1 for all production sites.
```

Comment:
@scottbuscemi is there proactive communication going to customers running upstreams with those old versions?

Reply:
It wasn't in the plans, but we could look into it.

@pwtyler Would we be able to detect this?

Reply:
Perhaps not with 100% coverage, but yes, I think we could do something.

Reply:
Adjusted the language slightly as I discovered this morning that site creation errors slightly more gracefully than I thought and site creation is not a dead-stop.